### PR TITLE
Adding new blog for rotating calipers

### PR DESCRIPTION
**Pull request by akashram14**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
how's this diff from that other blog

**Comment by akashram14** [comment]
the other blog was from 8yrs ago, with many broken links
found this recent blog on codeforces which explains quite well

**Comment by SansPapyrus683** [comment]
dyt this can replace it

**Comment by TheGamingMousse** [comment]
> dyt this can replace it

the other blog doesn't look like an actual resource/explanation while the one added in this PR does include a tutorial, so I think it's better (to my limited understanding)

File: content/5_Plat/Convex_Hull.mdx [hunk 1]
```diff
@@ -533,6 +533,11 @@ public class ConvexHull {
 		title="Rotating calipers technique and applications"
 		url="46162"
 	/>
+  <Resource
+    source="CF"
+    title="Rotating calipers technique and its applications"
```
Comment (ryanchou-dev) [inline content/5_Plat/Convex_Hull.mdx:538]: don't use the same title as another resource

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**